splitgraph/core/image.py

```diff
@@ -206,7 +211,12 @@ def lq_checkout(
 
         # It's easier to create the foreign tables from our side than to implement IMPORT FOREIGN SCHEMA by the FDW
         for table_name in self.get_tables():
-            if only_tables and table_name not in only_tables:
+            if (
+                only_tables
+                and table_name not in only_tables
```

Comment:
This might need to be `(only_tables and table_name not in only_tables) or table_name.startswith...`, I don't exactly remember the operator precedence rules

Reply:
I think those two are equivalent, but come to think of it the `table_name.startswith...` checks are not needed at all, as the upper/lower table should never make it to `splitgraph_meta.tables`